exist-core/src/test/xquery/xquery3/generate-id.xqm

```diff
@@ -0,0 +1,141 @@
+(:
+ : eXist-db Open Source Native XML Database
+ : Copyright (C) 2001 The eXist-db Authors
+ :
+ : info@exist-db.org
+ : http://www.exist-db.org
+ :
+ : This library is free software; you can redistribute it and/or
+ : modify it under the terms of the GNU Lesser General Public
+ : License as published by the Free Software Foundation; either
+ : version 2.1 of the License, or (at your option) any later version.
+ :
+ : This library is distributed in the hope that it will be useful,
+ : but WITHOUT ANY WARRANTY; without even the implied warranty of
+ : MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ : Lesser General Public License for more details.
+ :
+ : You should have received a copy of the GNU Lesser General Public
+ : License along with this library; if not, write to the Free Software
+ : Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ :)
+xquery version "3.1";
+
+module namespace gid = "http://exist-db.org/xquery/test/generate-id";
+
+declare namespace test = "http://exist-db.org/xquery/xqsuite";
+declare namespace xmldb = "http://exist-db.org/xquery/xmldb";
+
+declare
+    %test:setUp
+function gid:setup() {
+    xmldb:create-collection("/db", "generate-id-test"),
+    xmldb:store("/db/generate-id-test", "a.xml", <node>a</node>),
+    xmldb:store("/db/generate-id-test", "a.xml", <node>b</node>),
```

Comment:
is document name correct?

Reply:
Nope. I just pushed a commit to fix that’ll . Thanks